litmus/director/openebs-upgrade-jiva/test.yml

```diff
@@ -0,0 +1,216 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+  
+    ## Generating the testname for deployment
+    - include_tasks: /ansible-utils/create_testname.yml
+
+    ## RECORD START-OF-TEST IN LITMUS RESULT CR
+    - include_tasks: /ansible-utils/update_litmus_result_resource.yml
+      vars:
+        status: 'SOT'
+      
+    - set_fact:
+        director_url : "http://{{ director_ip }}:30380"
+
+    ## Getting the username
+    - name: Get username
+      shell: cat /etc/secret-volume/username
+      register: username
+
+    ## Getting the password.stdout     
+    - name: Get password
+      shell: cat /etc/secret-volume/password
+      register: password
+    
+    - block:
+        ## Install openebs using operator yaml
+        - name: Install openebs using operator yaml
+          shell: kubectl apply -f https://openebs.github.io/charts/openebs-operator-{{ openebs_current_version }}.yaml
```

Comment:
are we not supposed to use director to install?

Reply:
This test-case is for upgrade application that's why only the upgrade part is done from director in this PR.
For installation we have separate test cases. If it is good to have installation also director itself then i can make change in PR @AmitKumarDas 
cc: @harshshekhar15

Reply:
Yes @AmitKumarDas that should be the case. We should not deploy/install any application in the test itself.